### PR TITLE
[AF-402] Add rollbar_zaf_access_token

### PR DESCRIPTION
**Pull request by danielbreves**
✌️ 

### References
- JIRA: https://zendesk.atlassian.net/browse/AF-402

/cc @zendesk/vegemite

### CODE REVIEW

File: lib/zendesk_apps_support/assets/installed.js.erb [hunk 1]
```diff
@@ -14,4 +14,6 @@
   <% end %>
 }());
 
+ZendeskApps.rollbarAccessToken = "<%= rollbar_zaf_access_token %>";
```
Comment (sandlerr) [inline lib/zendesk_apps_support/assets/installed.js.erb]: nitpick: can this be an empty line in the template if the token is empty?

File: lib/zendesk_apps_support/assets/installed.js.erb [hunk 2]
```diff
@@ -14,4 +14,8 @@
   <% end %>
 }());
 
+<% unless rollbar_zaf_access_token.empty? %>
```
Comment (sandlerr) [inline lib/zendesk_apps_support/assets/installed.js.erb]: sorry, need to guard against this too:

```ruby
% irb
irb(main):001:0> nil.empty?
NoMethodError: undefined method `empty?' for nil:NilClass
	from (irb):1
	from /Users/rsandler/.rbenv/versions/2.4.0/bin/irb:11:in `<main>'
```

Comment (danielbreves) [inline lib/zendesk_apps_support/assets/installed.js.erb]: Why would it be `nil`?

Comment (sandlerr) [inline lib/zendesk_apps_support/assets/installed.js.erb]: if this code goes live server-side before the variable is available, or if running in zat

Comment (danielbreves) [inline lib/zendesk_apps_support/assets/installed.js.erb]: @sandlerr it defaults to an `empty` string [here](https://github.com/zendesk/zendesk_apps_support/pull/157/files#diff-67a579144c0a408fe3168e4a92bf4420R19).

Comment (sandlerr) [inline lib/zendesk_apps_support/assets/installed.js.erb]: oops didn't notice that

**Pull request merged**